### PR TITLE
Fixed Typographical Error in guide/english/kotlin/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by iamrajiv** [comment]
@thecodingaviator can you  review this

File: guide/english/kotlin/index.md [hunk 1]
```diff
@@ -107,11 +107,11 @@ If all goes well, the console window would open to show you the output.
 ![eclipse run app](https://kotlinlang.org/assets/images/tutorials/getting-started-eclipse/output.png)
 
 * ## Using the standalone compiler on the terminal
-If you are someone who prefers doing things in a more manual way and do not want to tie yourself down to an editor/IDE you might wanna use the Kotlin compiler.
+If you are someone who prefers doing things in a more manual way and does not want to tie yourself down to an editor/IDE you might wanna use the Kotlin compiler.
```
Comment (thecodingaviator) [inline guide/english/kotlin/index.md]: It was fine before

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by thecodingaviator** [comment]
@iamrajiv I've made a minor change on your behalf

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**